--- a/callback/Makefile.in
+++ b/callback/Makefile.in
@@ -51,6 +51,10 @@
 all-subdirs : force
 	cd @callback_subdir@ && $(MAKE) all
 
+vacall_r/libvacall.la: all-subdirs
+
+trampoline_r/libtrampoline.la: all-subdirs
+
 callback-libapi.lo : $(srcdir)/callback-libapi.c $(srcdir)/callback.h $(srcdir)/trampoline_r/trampoline_r.h ../config.h
 	$(LIBTOOL_COMPILE) $(CC) $(INCLUDES) -I$(srcdir)/trampoline_r $(CPPFLAGS) $(CFLAGS) -c $(srcdir)/callback-libapi.c
 

